packages/babel-polyfill/package.json

```diff
@@ -8,7 +8,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-polyfill",
   "main": "lib/index.js",
   "dependencies": {
-    "core-js": "^1.0.1",
+    "core-js": "^2.0.0",
```

Comment:
@thejameskyle I also thought that was part of the motivation and game plan for modularizing, but I just asked the other day in #development on slack how this situation would be handled and @sebmck seemed lukewarm at best about the idea of departing from the single version line approach.